Dockerfile

```diff
@@ -9,8 +9,31 @@ RUN export DEBIAN_FRONTEND="noninteractive" && apt-get update && \
     apt-get install -y --no-install-recommends \
       kicad kicad-libraries zip inkscape make git libmagickwand-dev \
       python3 python3-pip python3-wheel python3-setuptools inkscape \
-      libgraphicsmagick1-dev libmagickcore-dev openscad
+      libgraphicsmagick1-dev libmagickcore-dev openscad && \
+      rm -rf /var/lib/apt/lists/*
```

Comment:
I would remove this cleanup as it is no longer necessary.

Reply:
I think it is still useful. Everything from the `base` stage ends up in the final image. I'll verify that.

Reply:
Yup, removing the cleanup increases the image size by 27 MiB.

Reply:
Aah, my bad! I flipped the base and build it in my mind when reading the Dockerfile. Yeah, then it makes perfect sense and my other notes about throwing out stuff are pointless.